Tasks/Common/docker-common/dockercommandutils.ts

```diff
@@ -89,6 +91,38 @@ export async function getLayers(connection: ContainerConnection, imageId: string
     return layers.reverse();
 }
 
+export function getImageSize(layers: { [key: string]: string }[]): string {
+    let imageSize = 0;
+    for (const layer of layers) {
+        for (let key in layer) {
+            if (key.toLowerCase() === "size") {
+                const layerSize = extractSizeInBytes(layer[key]);
+                imageSize += layerSize;
+            }
+        }
+    }
+
+    return imageSize.toString() + "B";
+}
+
+export function extractSizeInBytes(size: string): number {
+    const sizeStringValue = size.match(matchPatternForSize);
+    if (sizeStringValue && sizeStringValue.length > 0) {
+        const sizeIntValue = parseFloat(sizeStringValue[0]);
+        const sizeUnit = size.substring(sizeIntValue.toString().length);
```

Comment:
you are not using "sizeStringValue[0]" for sizeIntValue.toString() because the "sizeIntValue.toString()" can have comma in it? if no reason, i would suggest to use "sizeStringValue[0]"